cli/emit.rs

```diff
@@ -236,13 +152,17 @@ pub fn get_ts_config_for_emit(
       let mut ts_config = TsConfig::new(json!({
         "allowJs": true,
         "allowSyntheticDefaultImports": true,
+        "checkJs": false,
         "experimentalDecorators": true,
         "incremental": true,
         "jsx": "react",
+        "jsxFactory": "React.createElement",
```

Comment:
Adding these here appears to cause issues when `"jsx"` is set to `"react-jsx"`.

Reply:
Hmm… I can’t remember why I added these :/ (but it’s late here)

Reply:
Ref: #15263